docs/examples.rst

```diff
@@ -8,7 +8,6 @@ Tutorials
 .. toctree::
    :maxdepth: 1
 
-   what_is_vagga
```

Comment:
I believe this is fine. Since it is actually a tutorial. And it's the simplest one.